lambdas/thumbnail/tests/test_thumbnail.py

```diff
@@ -109,6 +107,18 @@ def test_403():
             "pdf-page4-1024w.jpeg", None, [1024, 1450], 8, 200,
             marks=pytest.mark.poppler
         ),
+        pytest.param(
+            "pptx/in.pptx",
+            {"size": "w1024h768", "input": "pptx", "page": "1", "countPages": "true"},
+            "pptx/out-page1-1024w.jpeg", None, [1024, 1449], 2, 200,
+            marks=(pytest.mark.poppler, pytest.mark.loffice),
+        ),
+        pytest.param(
+            "pptx/in.pptx",
+            {"size": "w1024h768", "input": "pptx", "page": "2", "countPages": "true"},
+            "pptx/out-page2-1024w.jpeg", None, [1024, 1449], 2, 200,
+            marks=(pytest.mark.poppler, pytest.mark.loffice),
+        ),
```

Comment:
@akarve 
I used thumbs generated on the test stack as expected thumbs, but these tests fail for me locally, probably because of fonts. `MUMmer.pdf` tests also fail for me. Not sure if I could do something with it without running tests in the real lambda's Docker image, but I think that would be out of scope of this PR.

Reply:
But Lambda itself is linux; why can't you install the fonts locally? There are also some extended unit tests hidden behind a CLI switch that we should run.